spec/controllers/api/v1/routes_controller_spec.rb

```diff
@@ -97,18 +97,18 @@
   end
 
   describe 'GET show' do
-    it 'returns stops by OnestopID' do
+    it 'returns route by Onestop ID' do
       get :show, id: 'r-9q8y-richmond~dalycity~millbrae'
-      expect_json_types({
+      expect_json_types('route',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.